library/src/main/java/com/novoda/downloadmanager/DownloadManagerBuilder.java

```diff
@@ -12,6 +19,8 @@
  */
 public class DownloadManagerBuilder {
 
+    private static final int ANDROID_N = 24;
```

Comment:
my comments weren't against updates - they where against two separate things happening in 1 PR